lib/wasi-types/Cargo.toml

```diff
@@ -28,6 +28,7 @@ cfg-if = "1.0.0"
 anyhow = "1.0.66"
 byteorder = "1.3"
 time = "0.2"
+tracing = "0.1"
```

Comment:
Was this added by mistake?

I don't see it used, and it shouldn't be in this crate anyway.

Reply:
fixed